<launch>
    
    #Starting framegrabber node
    <node 
        pkg="rocket_tracker" 
        type="frame_grabber_node" 
        name="FRAMEGRABBER"
        required="true" 
        output="screen"/>

    #Starting image processor node
    <node 
        pkg="rocket_tracker" 
        type="image_processor_node" 
        name="IMAGEPROCESSOR"
        output="screen"/>

    #Starting evaluator node
    <node 
        pkg="rocket_tracker" 
        type="evaluator_node" 
        name="EVALUATOR"
        output="screen"/>
        
</launch>
